src/Tests/Indices/MappingManagement/GetMapping/GetMappingApiTest.cs

```diff
@@ -38,8 +38,6 @@ public class GetMappingApiTests : ApiIntegrationTestBase<IGetMappingResponse, IG
 
 		protected override void ExpectResponse(IGetMappingResponse response)
 		{
-			response.IsValid.Should().BeTrue();
```

Comment:
Looking at  [`ApiIntegrationTestBase`](https://github.com/elastic/elasticsearch-net/blob/2.x/src/Tests/Framework/ApiIntegrationTestBase.cs#L29), there is a check for a valid response in

```c#
[I] protected async Task ReturnsExpectedIsValid() =>
    await this.AssertOnAllResponses(r=>r.IsValid.Should().Be(this.ExpectIsValid));
```

so it seemed superfluous to check it again in `ExpectResponse` :smiley: 

I think we could remove the intermediate `GetRootObjectMappingWrapping` altogether and deserialize to `GetMappingResponse`:

1. if there are no indices, the `_mapping` endpoint returns `{}`
2. if there is a index with no mappings then `{index}/_mapping` returns `{ "index" : { "mappings": {} } }`